pkg/util/mount/mount_unsupported.go

```diff
@@ -47,7 +47,15 @@ func (mounter *Mounter) Unmount(target string) error {
 // GetMountRefs finds all other references to the device referenced
 // by mountPath; returns a list of paths.
 func GetMountRefs(mounter Interface, mountPath string) ([]string, error) {
-	return []string{}, nil
+	mps, err := mounter.List()
```

Comment:
The point of the unsupported mounter is that it is unsupported. I wouldn't expect implements here to make unit tests pass. Rather, I'd expect those unit tests not to be run on unsupported operating systems